Dockerfile

```diff
@@ -1,6 +1,8 @@
 # Use Docker in Docker stable version
+ARG ODTP_PATH
```

Comment:
Please rename this into `APP_PATH` since we already have `ODTP_PATH` used internally in the code with an other meaning.